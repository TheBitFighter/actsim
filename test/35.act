bool Reset;

defproc test2(bool w)
{
  bool a, b, c;
  
  w = a;
  prs {
    a | Reset => b-
    b => c-
    c => a-
  }
}

defproc test()
{
  bool q;
  test2 u(q);
}

bool qe;

Initialize {
  actions { Reset+;qe+ };
  actions { Reset- }
}
  
test t;    
