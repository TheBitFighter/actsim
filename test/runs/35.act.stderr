Identifier `qe' used, but not used in the design hierarchy selected.
FATAL: Error in initializer block?
